doc/ReleaseNotes/_ReleaseNotes.md

```diff
@@ -87,6 +84,17 @@
 * [Wasm] Add `PathIcon` support
 * [Wasm] Add UI Testing support through for `Uno.UI.Helpers.Automation.GetDependencyPropertyValue`\
 * Add automated UI testing support for every PR
+* Add support for `Windows.Phone.Devices.Notification.VibrationDevice` API on iOS, Android and WASM
```

Comment:
Are you going to merge this PR in the beta as well ?

Reply:
yes @jeromelaban  its already on the beta branch : https://github.com/nventive/Uno/commits/release/beta/Batman

The release/beta/Batman branch actually

Reply:
It is **not** part of current beta. I moved this out of 1.45.0.